django/thunderstore/frontend/models.py

```diff
@@ -12,6 +12,7 @@ class DynamicPlacement(ChoiceEnum):
     html_head = "html_head"
     html_body_beginning = "html_body_beginning"
     content_beginning = "content_beginning"
+    content_footer = "content_footer"
```

Comment:
Naming is semantically wrong, `content_footer` means it'd be the footer of the content area, whereas this has no relation to the content area. Flipping it to `footer_content` would work, or even just `footer`. Considering the footer is split to two parts though, perhaps `footer_top` or `footer_header` would make sense.

Reply:
Did this